### PR TITLE
API call to TFL and sample data files.

### PR DESCRIPTION
**Pull request by pablisch**
A simple API written in the frontend in App.js.
Data is filtered and mapped to return only arrivals that are due in the next 60 seconds, with selected data returned and sorted in order of arrival time.
Data samples have been collected and stored in sampleData folder for analysis and test use.

### CODE REVIEW

**Comment by pablisch** [comment]
It's so good not being able to push to the Maker's repo! 😀

File: frontend/src/App.js [hunk 1]
```diff
@@ -1,22 +1,44 @@
...
 import logo from './logo.svg';
 import './App.css';
 
 function App() {
+  const [tubeData, setTubeData] = useState([]);
+
+  const lines = "bakerloo,central,circle,district,hammersmith-city,jubilee,metropolitan,northern,piccadilly,victoria,waterloo-city";
+
+  useEffect(() => {
+    axios.get(`https://api.tfl.gov.uk/Line/${lines}/Arrivals?`)
```
Comment (sebwylleman) [inline frontend/src/App.js:12]: Could you let us know why you decided to use axios?

**Review by sebwylleman — APPROVED** [review]
Approving review with not many comments as this is the first iteration of the code.

**Review by tomcarmichael — APPROVED** [review]
Great stuff, passing tests!!

File: frontend/src/App.js (hunk 1 above)
Comment (pablisch) [reply frontend/src/App.js:12]: It seemed like a win-win over fetch. No apparent disadvantages but better error handling and browser compatibility. Easy to implement and you don't have to redirect chat gtp so much apparently.

**Pull request merged**